combat-trainer.lic

```diff
@@ -392,8 +392,25 @@ class LootProcess
   end
 
   def should_perform_ritual(prepare_consume)
```

Comment:
Should this be named `should_perform_ritual?` (note the '?')?

Reply:
yes